hikari/webhooks.py

```diff
@@ -126,8 +126,8 @@ async def execute(
             interaction webhooks.
 
             Additionally, flags this can only be set for interaction webhooks
-            and the only settable flag is EPHEMERAL; this field is just
-            ignored for non-interaction webhooks.
+            and the only settable flag is EPHEMERAL and SUPPRESS_NOTIFICATIONS;
+            this field is just ignored for non-interaction webhooks.
 
```

Comment:
This update seems wrong unless SUPPRESS_NOTIFICATIONS is actually just for interaction webhooks

Reply:
It worked everywhere I tested, which was in regular Message Creates, interaction responses and as plain webhooks. I suppose I should have paid better attention to the docstring 🤐